erpnext/projects/doctype/project/test_project.py

```diff
@@ -7,60 +7,127 @@
 test_records = frappe.get_test_records('Project')
 test_ignore = ["Sales Order"]
 
-from erpnext.projects.doctype.project_template.test_project_template import get_project_template, make_project_template
+from erpnext.projects.doctype.project_template.test_project_template import make_project_template
 from erpnext.projects.doctype.project.project import set_project_status
-
-from frappe.utils import getdate
+from erpnext.projects.doctype.task.test_task import create_task
+from frappe.utils import getdate, nowdate, add_days
 
 class TestProject(unittest.TestCase):
-	def test_project_with_template(self):
-		frappe.db.sql('delete from tabTask where project = "Test Project with Template"')
-		frappe.delete_doc('Project', 'Test Project with Template')
+	def test_project_with_template_having_no_parent_and_depend_tasks(self):
+		""" 
+		Test Action: Basic Test of a Project created from template. The template has a single task.
+		"""
+		frappe.db.sql('delete from tabTask where project = "Test Project with Templ - no parent and dependend tasks"')
```

Comment:
Naming consistency with Task and Project naming. Use title case for the entire project name. `Test Project With Template - No Parent and Dependant Tasks`